### ISSUE TITLE
DataTable: Tabbing though cells throws an error when combined with a selectable checkbox column

### ISSUE THREAD

**Issue by admin-nexgen** [primefaces/primevue#3202]
### Describe the bug

I am trying to create an editible table where the first row is used to filter the data, but I also want to be able to tab into the cells that follow after the top row filter.

Getting an error when tabbing though a filter row when combined with a checkbox column. 

It works when I don't have the checkbox column, but it also doesn't tab into the editable row, which I would expect to be able to do somehow so that you can seamlessly edit the whole table even if it has a filter row, or even a frozen row.

When you tab a few times starting at the filter row, it eventually throws an error once it gets into the checkbox row and tries to find the next control to tab into, which should be the editable table, but it doesn't work.

TypeError
Cannot read properties of null (reading 'nextElementSibling')

### Reproducer

https://codesandbox.io/s/dazzling-herschel-gnoufo?file=/src/DataTableEditDemo.vue

### PrimeVue version

3.18.1

### Vue version

3.x

### Language

TypeScript

### Build / Runtime

Vite

### Steps to reproduce the behavior

1. Click into the first filter cell and tab through the fields until you get an error.
2. If you add an extra frozen row, it also breaks in the same way (we use the frozen row as a way to add a new row into the table)

### Expected behavior

Should not get an error when tabbing though the filter fields and ideally should have a way to just carry on tabbing into the rest of the editable table.

There should also be a way to disable tabbing on the filter icons if possible and only target the text fields

**Comment by tugcekucukoglu** [comment]
Could you test it with the latest version? If the problem still persists, please create a new issue.
